### ISSUE TITLE
Firefox extension

### ISSUE THREAD

**Issue by man2machine** [alganzory/HumanBlur#10]
Assalamu alaikum

It would be great if you could publish the latest version of your extension to the Firefox extension store!

Jazak Allahu Khair.

**Comment by alganzory** [comment]
Walaikum Assalam, 

Yes, I am working on this too, soon inshallah. 

JazakAllahu Khairan

**Comment by man2machine** [comment]
I use the firefox browser so this would be quite helpful! I would recommend keeping it in the same repo, but then organizing the repo so that there is a chrome folder and a firefox folder. Otherwise it would be hard to deal with changes that affect both in the long run.

**Comment by anonfaded** [comment]
As Salam o alaikum
Brother any updates about the firefox extension?
When will that be available :)

**Comment by LinuxJava7** [comment]
Where will the work be done on this extension? Here or on a different repo?

**Comment by alganzory** [comment]
> Where will the work be done on this extension? Here or on a different repo?

Probably here on a separate branch

**Comment by Nya0** [comment]
when is eta?

**Comment by alganzory** [comment]
@Nya0 Within a day or two, I just posted about it on [twitter ](https://twitter.com/HaramBlur/status/1737069703158226991) today

**Comment by alganzory** [comment]
HaramBlur for Firefox is finally here

[Download link](https://addons.mozilla.org/en-US/firefox/addon/haramblur/ )
[Twitter post](https://twitter.com/HaramBlur/status/1737897690057572436)

**Comment by LinuxJava7** [comment]
Unfortunately it doesn't seem to work on Firefox for Android
But the desktop one works great

**Comment by alganzory** [comment]
@LinuxJava7 yeah I didn't really test it for Android, I should probably disable the android version until I've tuned it for it I think

**Comment by LinuxJava7** [comment]
Now that I checked again, it doesn't work on Kiwi Browser either (it has desktop chromium extensions)

**Comment by alganzory** [comment]
Huh, I remember it working on Kiwi, anyway, can you please create a separate issue for this so I can keep track of it?

**Comment by LinuxJava7** [comment]
Should I make an issue about firefox on android?
